magento/magento2
# 8003: Using System Value for Base Currency Results in Config Error.

### Description
Fix for "Using System Value for Base Currency Results in Config Error".

### Fixed Issues (if relevant)
1. magento/magento2#8003: Using System Value for Base Currency Results in Config Error

### Manual testing scenarios
1. Log into Admin.
2. Navigate to Stores > Settings > Configuration > General > Currency Setup.
3. Select Use system value for Base Currency.
4. Click Save Config if necessary and flush the cache.
5. Navigate to Stores > Currency > Currency Rates.
6. Try Importing Currency Rates via any of the three services.
7. Check currency rates are successfully imported for Allowed Currencies.

### Contribution checklist
